looprpc/client.proto

```diff
@@ -580,6 +592,13 @@ message QuoteRequest {
     Optional route hints to reach the destination through private channels.
     */
     repeated RouteHint loop_in_route_hints = 6;
+
+    /*
+    Private indicates whether the destination node should be considered
+    private. In which case, loop will generate hophints to assist with
```

Comment:
Still not super happy with the wording here. "destination node should be considered private" likely doesn't make a lot of sense to the end-user. "Whether to include hop hints" would read better imo.